doc/source/reference/routines.other.rst

```diff
@@ -35,3 +36,26 @@ NumPy version comparison
    :toctree: generated/
 
    lib.NumpyVersion
+
+NumPy header files location
+----------------------------
+.. autosummary::
+   :toctree: generated/
+   
+   get_include
+
+Deprecation warning
```

Comment:
Not a huge fan of this title, but I guess better than not having any reference at all.

Reply:
I had initially thought of _Warning Generator_ for the title here, but that seemed ambiguous. It may not be intuitive for a beginner looking for ```deprecate```. What are we looking for in a good title?

Reply:
How is _Handling Old Functions in NumPy_ for the title?